superset/views/core.py

```diff
@@ -96,6 +96,13 @@ def get_datasource_access_error_msg(datasource_name):
               '`all_datasource_access` permission', name=datasource_name)
 
 
+def get_dashboard_access_error_msg(dashboard_title):
```

Comment:
Let's put free-floating functions in `utils.py` - this module is already enormous.